plugins/mcreator-localization/lang/texts.properties

```diff
@@ -3161,6 +3161,7 @@ dialog.tools.inject_tags.tag.entities.minecraft.beehive_inhabitors=Entities in t
 dialog.tools.inject_tags.tag.entities.minecraft.impact_projectiles=Entities in this tag group can activate targets.
 dialog.tools.inject_tags.tag.entities.minecraft.raiders=Entities in this tag participate in raids.
 dialog.tools.inject_tags.tag.entities.minecraft.skeletons=Entities in this tag can kill creepers and drop music discs.
+dialog.tools.inject_tags.tag.entities.minecraft.powder_snow_walkable_mobs=Entities in this tag walk on snow powder blocks.
```

Comment:
`Entities in this tag can walk on snow powder blocks.`

Otherwise it sounds they can only walk on show power blocks and not on other blocks

Reply:
I only say they are allowed to walk on snowpowder blocks. I don't say they can only walk on snowpowder blocks.

Reply:
I just said how it _could_ sound to the user and thus suggested alternative wording

Reply:
Is `Entities in this tag are allowed to walk on snow powder blocks.` a better wording?

Reply:
Could be.

Or `Entities in this tag can walk on top of snow power blocks` or `Entities in this tag are allowed to walk on top of snow powder blocks`

Reply:
I'm not sure how adding "on top of" to the original text makes it clearer, but ok.

Reply:
... Ok, my bad. I just noticed I didn't write the "can". I thought I wrote `Entities in this tag can walk on snow powder blocks.`, so this is why I wasn't understanding your incomprehension. Sorry about that. I'll add the `can`